Rakefile

```diff
@@ -11,6 +11,15 @@ require 'rspec'
 require 'rspec/core/rake_task'
 require 'cucumber/rake/task'
 
+def rails_template_command
+  require "rails"
+  if Rails.version.to_f >= 5.1
```

Comment:
I think `if Rails.version.to_f >= 5.0` is better, because ` "rails:template"` was deprecated on Rails 5.0.